ui-react/packages/atlasmap/src/Views/ColumnMapperView/Actions/EnableJavaClassAction.tsx

```diff
@@ -1,33 +1,38 @@
 import React, { FunctionComponent } from "react";
-
 import { Button, Tooltip } from "@patternfly/react-core";
 import { AddCircleOIcon } from "@patternfly/react-icons";
 
 export interface IEnableJavaClassActionProps {
-  id: string;
-  onClick: () => void;
+  isSource: boolean;
```

Comment:
No need to pass this function this info since it’s used just to pass it back again to the onCustomClassSearch callback. Just create the appropriate closure where this is used